ios/src/Data-Bridging/ACPExtensionEvent+ReactNative.m

```diff
@@ -24,7 +24,7 @@ + (ACPExtensionEvent *)eventFromDictionary: (nonnull NSDictionary *) dict {
     NSString *type = [dict objectForKey:EVENT_TYPE_KEY];
     NSString *source = [dict objectForKey:EVENT_SOURCE_KEY];
 
-    if (name && type && source && [[dict objectForKey:EVENT_DATA_KEY] isKindOfClass:[NSDictionary class]]) {
+    if (name && type && source && ([[dict objectForKey:EVENT_DATA_KEY] isKindOfClass:[NSDictionary class]] || ![dict objectForKey:EVENT_DATA_KEY])) {
```

Comment:
This logic was originally incorrect, which should accept events that do not contain any event data.